src/ds/features/auto-exposure-gain-limit-feature.h

```diff
@@ -0,0 +1,27 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#pragma once
+
+
+#include <src/feature-interface.h>
+
+#include <memory>
+
+namespace librealsense {
+
+class synthetic_sensor;
+class hw_monitor;
+
+class auto_exposure_gain_limit_feature : public feature_interface
```

Comment:
I think we need to seperate it to a `range_limit_featrue` that is inherited by `auto_exposure_limit_feature` and `gain_limit_feature`.
Maybe we will need later for other ranges as well - brightness, hue, contrast etc...